src/features/profile/pages/NotificationSettings.tsx

```diff
@@ -0,0 +1,84 @@
+import { t } from '@lingui/macro'
+import React, { useState } from 'react'
+import styled from 'styled-components/native'
+
+import { _ } from 'libs/i18n'
+import { ButtonPrimary } from 'ui/components/buttons/ButtonPrimary'
+import FilterSwitch from 'ui/components/FilterSwitch'
+import { ColorsEnum, getSpacing, Spacer, Typo } from 'ui/theme'
+
+import { ProfileHeaderWithNavigation } from '../components/ProfileHeaderWithNavigation'
+import { ProfileContainer, Separator } from '../components/reusables'
+import { SectionRow } from '../components/SectionRow'
+
+export function NotificationSettings() {
+  const [allowEmails, setAllowEmails] = useState(false)
+  const [allowPush, setAllowPush] = useState(false)
+
+  return (
+    <React.Fragment>
+      <ProfileHeaderWithNavigation title={_(t`Notifications`)} />
+      <Spacer.Column numberOfSpaces={6} />
+      <ProfileContainer>
+        <Typo.Body color={ColorsEnum.BLACK}>
+          {_(t`Reste informé des actualités du pass Culture et ne rate aucun de nos bons plans.`)}
+        </Typo.Body>
+        <Spacer.Column numberOfSpaces={4} />
+        <Separator />
+        <Line>
+          <SettingExplanation>
+            {_(
+              t`Je veux recevoir les actualités et les meilleures offres du pass Culture par e-mail.`
```

Comment:
```suggestion
              t`Je veux recevoir les actualités et les meilleures offres du pass Culture par e\u2011mail.`
```
Pour éviter un retour à la ligne sur le tiret comme sur la capture